### PR TITLE
ADCM-2972 Refactor audit

### PR DESCRIPTION
**Pull request by winmasta**

### CODE REVIEW

**Review by SofiaTal — APPROVED** [review]

**Comment by winmasta** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

**Comment by winmasta** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

**Comment by winmasta** [comment]
Bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

**Comment by winmasta** [comment]
Bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

File: python/audit/tests/test_host.py [hunk 1]
```diff
@@ -38,7 +38,30 @@ def setUp(self) -> None:
             prototype=provider_prototype,
         )
         self.fqdn = "test_fqdn"
-        self.audit_operation_create_host = AUDIT_OPERATION_MAP["HostListProvider"]["POST"]
+
+    def check_host_created(self, log: AuditLog, res: Response):
+        assert log.audit_object.object_id == res.data["id"]
+        assert log.audit_object.object_name == self.fqdn
+        assert log.audit_object.object_type == AuditObjectType.Host.value
```
Comment (a-alferov) [inline python/audit/tests/test_host.py:45]: I think `.value` is redundant

Comment (winmasta) [reply python/audit/tests/test_host.py]: Fixed

**Review by a-alferov — APPROVED** [review]

**Comment by winmasta** [comment]
Bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

**Comment by winmasta** [comment]
Bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

**Comment by winmasta** [comment]
Bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

**Pull request merged**